Gemfile

```diff
@@ -21,8 +21,15 @@ gem "bootsnap", ">= 1.4.2", require: false
 
 group :development, :test do
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  gem "factory_bot_rails"
   gem "rubocop", "~> 0.74.0", require: false
   gem "rubocop-rails", "~> 2.3.2", require: false
+  %w[rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
dùng 1 dòng gem "rspec-rails" là được r, ko cần dài dòng như này đâu

Reply:
Em đã sử dụng chỉ sử dụng gem "rspec-rails" nhưng bị gặp lỗi nên em đã sửa như trên ạ.
Em gặp lỗi như trong bài sau và đã sửa theo ạ. 
http://jessehouse.com/blog/2019/06/19/actionview-template-error-wrong-number-of-arguments-given-2/

Reply:
@phudv-0958 thử chạy gem 'rspec-rails', '~> 4.0' vẫn lỗi à e

Reply:
@ngocvt-0484 chị ơi, ái  'rspec-rails', '~> 4.0' , báo không tìm thấy a. Còn gem "rspec-rails", git: "https://github.com/rspec/rspec-rails", branch: "4-0-maintenance" chạy ok ạ.